python/tvm/relay/op/strategy/x86.py

```diff
@@ -50,7 +50,7 @@ def schedule_reduce_cpu(attrs, outs, target):
 def schedule_concatenate_cpu(attrs, outs, target):
     """schedule concatenate op for x86"""
     with target:
-        return topi.x86.schedule_concatenate(outs)
+        return topi.transform.schedule_concat(outs)
```

Comment:
We should remove `schedule_concatenate_cpu` here and instead move the call to `topi.transform.schedule_concat` under `strategy/generic.py`. See for example how `cumsum` op is implemented: https://github.com/apache/tvm/blob/45af5c73a9ce07bcd50d36150680259b719dc8c7/python/tvm/relay/op/strategy/generic.py#L1664-L1672

Reply:
Done.